src/stumpy_png/chunk.cr

```diff
@@ -14,7 +14,7 @@ module StumpyPNG
       data = slice[4, slice.size - 8]
 
       expected_crc = Digest::CRC32.checksum(slice[0, slice.size - 4])
-      raise "Incorrect checksum" if crc != expected_crc
```

Comment:
I like the `if` better since the exception is caused by the checksums not being equal

Reply:
fwiw `if !=` is logically equal to `unless ==`.